db/migrate/20150331170121_create_beers.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBeers < ActiveRecord::Migration
+  def change
+    create_table :beers do |t|
+      t.string :name, null: false
+      t.integer :abv
+      t.integer :ibu
+      t.text :description
+      t.integer :style_id, null: false
+      t.integer :user_id, null: false
```

Comment:
If you're going to be searching for beers by user or beers by style, add an index on each foreign key to speed up the queries.